src/main/java/Task.java

```diff
@@ -0,0 +1,68 @@
+public class Task {
+    //Instantiate
+
+    private static final String LINE = "____________________________________________________________";
+    private static final String BLANK = "    ";
+    protected String taskName;
+    protected boolean isDone;
+
+    public Task(String taskName) {
+        this.taskName = taskName;
+        this.isDone = false;
+    }
+    public void setDone() {
+        isDone = true;
+    }
+    public void setNotDone() {
+        isDone = false;
+    }
+    public String getTaskName() {
+        return taskName;
+    }
+
+    public void printTaskType() {
+
+    };
+    public void printTask() {
+        System.out.print("[");
+        printTaskType();
+        System.out.print("][");
+        if (this.isDone) {
+            System.out.print("X");
+        } else {
+            System.out.print(" ");
+        }
+        System.out.println("] " + this.getTaskName());
+    }
+    public void printAddTask(int taskNumber) {
+        System.out.println(BLANK + LINE);
+        System.out.println(BLANK + "Added to list:");
+        System.out.print(BLANK);
+        printTask();
+        System.out.println(BLANK + "Number of tasks: " + (taskNumber + 1));
+        System.out.println(BLANK + LINE);
+    }
+    public void printDeleteTask(int numberOfTask) {
+        System.out.println(BLANK + LINE);
+        System.out.println(BLANK + "Deleting from list:");
+        System.out.print(BLANK);
+        printTask();
+        System.out.println(BLANK + "Number of tasks: " + (numberOfTask - 1));
+        System.out.println(BLANK + LINE);
+    }
+    public void printMarkedTask() {
+        System.out.println(BLANK + LINE);
+        System.out.println(BLANK + "Nice! I've marked this task as done:");
+        System.out.print(BLANK);
+        printTask();
+        System.out.println(BLANK + LINE);
```

Comment:
Consider other ways of printing here as well, instead of having too many lines of System.out.println(). Maybe a UI class to handle the printing of all outputs? :)